gamemaker.py

```diff
@@ -30,4 +32,52 @@ def logout(self):
         return return_value
 
     def display_menu(self):
-        return "Options\n\nlog out\ndisplay status\n"
+        return "Options\n\nmake team\n\nlog out\ndisplay status\n"
+
+    def make_team(self, input):
+        input = input.split(" ")
+        if len(input) == 2:
+            name = input[0]
+            password = input[1]
+            self.database.add_team(team.Team(name, password, self.database))
+            ret_string = "Added " + name
+        else:
+            ret_string = "Invalid input!"
+        return ret_string
+
+    def edit_team(self, input):
+        found = False
+        input = input.split(" ")
+        if len(input) == 3:
+            list = self.database.get_teams()
+            for teams in list:
+                if teams.username == input[0]:
+                    found = True
+                    teams.username = input[1]
+                    teams.password = input[2]
+                    ret_string = "" + input[0] + " changed to " + input[1] + " with password " + input[2]
+        else:
+            ret_string = "Invalid input!"
+            found = True
+        if not found:
+            ret_string = "Could not find that team!"
+        return ret_string
+
+    def set_penalties(self, input):
+        exception = False
+        input = input.split(" ")
+        if len(input) == 2:
+            try:
+                time = int(input[0])
+                guess = int(input[1])
+            except ValueError:
+                exception = True
+            if not exception and time > 0 and guess > 0:
+                self.database.time_penalty = time
```

Comment:
I believe there are getters and setters for penalties in the db. If not, they should be added.